### ISSUE TITLE
Rocket Player 4.2.52 from October 2016 stops working since SLS update to 1.5.2 last week

### ISSUE THREAD

**Issue by MartinL0815** [tgwizard/sls#333]
I use Rocket Player (currently 4.2.52 from October) for locally stored music.
And I scrobble the played tracks using SLS.

Today I realized, that scrobbling works for one track only. Then I restart RP and it works again, for a track.

Since SLS was just updated to 1.5.2 and RP in October I write at first to you to describe this "bug".

What kind of information can I provide to help to stop this behavior?

**Comment by HumbleBeeBumbleBee** [comment]
This is a known issue.
https://github.com/tgwizard/sls/blob/master/TroubleShooting.md#know-issue-compatibility-vs-structure

**Comment by MartinL0815** [comment]
Ok …

> The current Queue method works for now.

So, can I change something to get to work at all - with Rocket Player?

Or should I try to contact Justin Tipton from the RP Team to describe this problem, share information?

**Comment by HumbleBeeBumbleBee** [comment]
Yes. If he adds the broadcast Track state as playback completed it should scribble track information for a single repeat track.

**Comment by MartinL0815** [comment]
I used the Google+ Community of Rocket Player to report the issue and to connect this with the that:

> [g+ RP community message ](https://plus.google.com/+MartinLemburgBerlinDE/posts/YnP8nQ57sHo) https://plus.google.com/+MartinLemburgBerlinDE/posts/YnP8nQ57sHo

**Comment by HumbleBeeBumbleBee** [comment]
@MartinL0815 Could you please explain the problem of Rocket Player not working at all for your device?

**Comment by HumbleBeeBumbleBee** [comment]
@MartinL0815 Have you tried enabling "OnGoing" (Phone Notifications)?

**Comment by HumbleBeeBumbleBee** [comment]
> So, can I change something to get to work at all - with Rocket Player?

Yes please try enabling "OnGoing" (Phone Notifications).

I can help you debug it if need be.

**Comment by MartinL0815** [comment]
Sorry, but I do not know the term "ongoing" in relations to notifications.

Speculating, I would say it is about always staying notifications with high priority?

**Comment by HumbleBeeBumbleBee** [comment]
Something like this, yes.

**Comment by HumbleBeeBumbleBee** [comment]
If you have the time, I can help you debug it.

**Comment by HumbleBeeBumbleBee** [comment]
@MartinL0815  Any progress with this issue? Is there anything I can do to help?

**Comment by aniol** [comment]
I have the same issue since 2016: Simple Last.fm Scrobbler (Beta) is only scrobbling when it's open on the screen.

**Comment by HumbleBeeBumbleBee** [comment]
What version of Android please?

**Comment by aniol** [comment]
Android 6.0 with EMUI 4.1.1.

**Comment by HumbleBeeBumbleBee** [comment]
So you tried enabling "OnGoing"?

**Comment by aniol** [comment]
Yes, "OnGoing" is enabled.

**Comment by HumbleBeeBumbleBee** [comment]
@aniol Has anything changed since we last spoke? I need to check the information Rocket Player shares to SLS.

**Comment by aniol** [comment]
No. Nothing changed with the new update (1.5.3) of the app.

**Comment by HumbleBeeBumbleBee** [comment]
I assume you changed the settings in Rocket Player to support SLS?

**Comment by aniol** [comment]
![img_20170212_171846](https://cloud.githubusercontent.com/assets/2990117/22864591/77837660-f154-11e6-83f8-4ffc4fe4fdda.png)
Yes, I selected Simple Last.fm in Rocket Player settings.
